src/nagini_translation/lib/jvmaccess.py

```diff
@@ -14,7 +14,7 @@ class JVM:
 
     def __init__(self, classpath: str):
         jpype.startJVM(jpype.getDefaultJVMPath(),
-                       '-Djava.class.path=' + classpath, '-Xss128m')
+                       '-Djava.class.path=' + classpath, '-Xss32m')
```

Comment:
I had issues with stack size in the command line (calling silicon), tricky to get the right value. If we have issues with stack size while running nagini, will the user have a good error message?

Reply:
No, they'll just get a stack trace. But 32 MB is more than enough for all cases I've ever tried, and I sometimes get weird crashes with larger values, that's why I changed it.